ovirt_imageio/client/_ui.py

```diff
@@ -53,9 +59,21 @@ def draw(self, value, transferred, phase=None, last=False):
         return line.ljust(self._width, " ") + end
 
 
+class JsonFormat(OutputFormat):
+
+    def draw(self, value, transferred, phase=None, last=False):
+        return json.dumps({
+            'transferred': transferred,
+            'size': self.size,
+            'elapsed': self.elapsed,
+            'description': phase or "",
+        }) + '\n'
```

Comment:
This works but using 2 lines will be nicer:

   content = json.dumps(...)
   return content + "\n"